### PR TITLE
Feat implement convert cloud convert file php

### PR DESCRIPTION
**Pull request by verenaroe**
Part of [appwrite/appwrite#1933](https://github.com/appwrite/appwrite/issues/1933)

#### Initial Storage
grafik

#### Execution input
grafik

#### Execution log
grafik

#### Execution output
grafik

#### Final storage
grafik

### CODE REVIEW

File: php/convert-cloud-convert-file/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+## 🚀 Building and Packaging
+
+To package this example as a cloud function, create a .tar file of the code and deploy it on the Appwrite dashboard.
+
+#### Create .tar file
+
+```bash
+$ cd php/convert-cloud-convert-file
+$ composer install
+$ tar -zcvf code.tar.gz .
```
Comment (christyjacob4) [inline php/convert-cloud-convert-file/README.md:28]:
```suggestion
$ cd ..
$ tar -zcvf code.tar.gz  convert-cloud-convert-file
```

File: php/convert-cloud-convert-file/README.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+If you want to use the Appwrite CLI no .tar packaging is needed. You can use the `createTag` command. Use 
+
+```bash
+$ appwrite functions createTag help
+```
+to learn more about the command. You have to install the Appwrite CLI first.
+
+
+## 🎯 Schedule
```
Comment (christyjacob4) [inline php/convert-cloud-convert-file/README.md:47]:
```suggestion
## 🎯 Trigger
```

Comment (christyjacob4) [reply php/convert-cloud-convert-file/README.md:47]: Mention in the description that this function uses a CRON schedule as a trigger.

**Review by christyjacob4 — CHANGES_REQUESTED** [review]
Please address the comments

**Comment by verenaroe** [comment]
I addressed all your comments

**Comment by christyjacob4** [comment]
THANK YOU! All changes merged :partying_face:

Please reach out to me on discord to claim your Hacktoberfest swag! 

As a way of saying thank you, we would also love to invite you to join the Appwrite organization on GitHub. Please share your GitHub username with us on Discord so that we can add you as a contributor to our organization.

Once added, you can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite membership badge will show up on your personal GitHub profile.

**Pull request merged**